feg/gateway/services/s8_proxy/client_api.go

```diff
@@ -62,7 +65,15 @@ func SendEcho(req *protos.EchoRequest) (*protos.EchoResponse, error) {
 }
 
 func getS8ProxyClient() (*s8ProxyClient, error) {
-	conn, err := registry.GetConnection(registry.S8_PROXY)
+	var (
+		conn *grpc.ClientConn
+		err  error
+	)
+	if util.GetEnvBool("USE_REMOTE_S8_PROXY") {
```

Comment:
You may also use util.GetEnvBool("USE_REMOTE_S8_PROXY", true) if you want it default to remote

Reply:
But then when `USE_REMOTE_S8_PROXY` is missing it will be defaulted to true. 
Isn't it kind of counter intuitive?